core/src/trezor/lvglui/i18n/locales/zh_hk.py

```
@@ -820,7 +820,7 @@
     "選擇帳戶",
     "前往帳戶",
     "輸入格式錯誤",
-    "ERC-20 代幣在 Ethereum 網路上發行。這意味著可以使用 ETH 錢包地址接收任何 ERC-20 代幣。",
+    "當前地址是一個 EVM 網絡地址。你可同步管理其他 EVM 兼容網絡（例如 Ethereum、BNB Chain、Polygon、Arbitrum One、Avalanche等）上的資產。",
```

Ruff (0.8.2)

String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)